code/controllers/subsystem/processing/overlays.dm

```diff
@@ -0,0 +1,115 @@
+var/datum/subsystem/processing/overlays/SSoverlays
+
+/datum/subsystem/processing/overlays
+	name = "Overlays"
+	flags = SS_TICKER|SS_NO_INIT|SS_FIRE_IN_LOBBY|SS_KEEP_TIMING
```

Comment:
keep_timing with a 1 tick ticker subsystem makes no sense (it can't run it earlier if it was delayed last fire if it runs every tick anyways) and has extra overhead.